sdk/include/c++/stlport/stl/config/_reactos.h

```diff
@@ -302,6 +302,9 @@
 // Calling convention
 #define _STLP_CALL __cdecl
 
+// Always build multithreaded code
+#define _STLP_THREADS 1
```

Comment:
2 places set it to `1`, but a value looks superfluous/misleading to me.
```suggestion
#define _STLP_THREADS
```